R-packages/covidcast/R/covidcast.R

```diff
@@ -529,58 +532,59 @@ covidcast_days <- function(data_source, signal, start_day, end_day, geo_type,
   for (i in seq(ndays + 1)) {
     query_day <- start_day + i - 1
     day_str <- date_to_string(query_day)
-    dat[[i]] <- covidcast(data_source = data_source,
-                          signal = signal,
-                          time_type = "day",
-                          geo_type = geo_type,
-                          time_values = day_str,
-                          geo_value = geo_value,
-                          as_of = as_of,
-                          issues = issues,
-                          lag = lag)
+    res <- covidcast(data_source = data_source,
```

Comment:
Not really keen on the variable name "res" but I don't have a better, more descriptive suggestion off the top of my head.

Reply:
How about `response`?